packages/workflow/src/records/state-transitions.ts

```diff
@@ -269,6 +271,46 @@ export async function toUpdated(
   return updatedRecord
 }
 
+export async function toViewed(
+  record: ValidRecord,
+  user: IUserModelData,
+  office: Location
+) {
```

Comment:
We know that our returned record from this function will be a `validRecord` so we can coerce it be so if needed